.stylelintrc.json

```diff
@@ -21,7 +21,7 @@
         "ignoreProperties": ["composes"]
       }
     ],
-    "custom-property-pattern": "g|color|font|elevation|space|opacity|motion|font-.+",
+    "custom-property-pattern": "g|color|font|elevation|space|opacity|motion|font-.+|size",
```

Comment:
yeah I'd avoid this size one. as we don't have size categories yet!

Reply:
Can we support a size category? 

@diyorbek are these size tokens provided by design